travelogue/app/views/sessions/new.html.erb

```diff
@@ -1,13 +1,20 @@
-You're on Views/Sessions/new.html.erb
-<br>
-
-Login
-<%= form_for @user, url: "/sessions", method: "post" do |f| %>
-	<div>
-		<%= f.text_field :email, placeholder: "Email" %>
-	</div>
-	<div>
-		<%= f.password_field :password, placeholder: "Password" %>
-	</div>
-	<%= f.submit "Sign In" %>
-<% end %> 
\ No newline at end of file
+<%= form_for @user, :html => {:class => "form-horizontal center-block"} do |f| %>
+  <h1 class="form-group">Login</h1>
+  <div class="form-group">
+    <%= f.label :email, "email", class: " control-label" %>
+    <div>
+      <%= f.text_field :email, class: "form-control" %>
+    </div>
+  </div>
+  <div class="form-group">
+    <div>
```

Comment:
Is this inner div for anything?  I see it on each of these but they're all also wrapped in an outer div; so I'm not sure.